install/yamls/ccruntime-peer-pods-in-pod.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cloud-api-adaptor-deployment
+  namespace: peer-pods
```

Comment:
Better to use the same namespace as the CRD - `confidential-containers-system`

Reply:
Yes i wondered about it, i'd also prefer it.. because  ATM it's not really manged by the operator, it's installed "on the side" with the runtime i didn't do so, If that's not a problem I'll change that

Reply:
Let's change it to use `confidential-containers-system` to ensure all peer-pods resources are in the same namespace.